bootstrap/021_external_functions.sql

```diff
@@ -376,8 +376,16 @@ BEGIN
       INSERT (key, value)
       VALUES (source.key, source.value);
 
-    -- Start the user limits task (we know that Sundeck is linked)
-    CALL internal.start_user_limits_task();
+    -- Start the USER_LIMITS_TASK if the task exists
+    begin
+        show tasks in schema tasks;
+        let has_task text := (select any_value("name") from table(result_scan(last_query_id())) where "name" = 'USER_LIMITS_CHECK');
+        if (:has_task is not null) then
+            -- If the user is setting up the native app via the Streamlit UI, the task exists but has not been started.
+            -- If the user is setting up the native app via Sundeck, the task does not yet exist and will be started after it is created.
+            CALL internal.start_user_limits_task();
```

Comment:
This is confusing to me because I thought task was not there  and that's why there was a failure.
`internal.start_user_limits_task()` does not create it, it just starts it ...